Client/appsettings.json

```diff
@@ -0,0 +1,3 @@
+{
+	"ConnectionString": "https://localhost:44354/api/"
```

Comment:
Connection string is used with DB.
This string is called Url.
A possible name to this field would be "ServerUrl"